_posts/2014-05-29-touristic-autism_google-map.markdown

```diff
@@ -4,25 +4,24 @@ title: Touristic Autism-friendly Spots App
 permalink: touristic-autism_google-map
 ---
 
-# Show All Places in a Google Map
+# Mostrar todos os lugares em um mapa do Google
 
-*Created by Myriam Leggieri, [@iammyr](https://twitter.com/iammyr)*
-*for [Rails Girls Galway](https://github.com/RailsGirlsGalway)*
-The basic guides that have been merged and adapted are the [Ruby on Rails Tutorial](http://www.railstutorial.org/book), the [basic RailsGirls app](http://guides.railsgirls.com/app/) and the tutorials for [creating thumbnails](http://guides.railsgirls.com/thumbnails), [authenticating users](http://guides.railsgirls.com/devise/), [adding design](http://guides.railsgirls.com/design), [deploying to OpenShift](http://guides.railsgirls.com/openshift/) and [adding comments](http://guides.railsgirls.com/commenting).
+*Criado por Myriam Leggieri, [@iammyr](https://twitter.com/iammyr)*
 
+*para [Rails Girls Galway](https://github.com/RailsGirlsGalway)*
+Os guias básicos que foram mesclados e adaptados são os [Ruby on Rails Tutorial](http://www.railstutorial.org/book), [basic RailsGirls app](http://guides.railsgirls.com/app/) e os tutoriais são [creating thumbnails](http://guides.railsgirls.com/thumbnails), [authenticating users](http://guides.railsgirls.com/devise/), [adding design](http://guides.railsgirls.com/design), [deploying to OpenShift](http://guides.railsgirls.com/openshift/) e [adding comments](http://guides.railsgirls.com/commenting).
 
-We need to install a piece of software to let us display and interact with Google Maps.
+Nós precisamos instalar um software que nos permita exibir e interagir com o Google Maps.
 
-Open `Gemfile` in the project directory using your text editor and add
+Abra `Gemfile` na sua pasta do projeto usando seu editor de texto e adicione:
 
-{% highlight ruby %}
+```
```

Comment:
Aqui geralmente deixamos a sintaxe de highlight mesmo, como fala no guia original: https://github.com/railsgirls/railsgirls.github.io#pygments-and-code-highlighting